script/constructron.lua

```diff
@@ -503,28 +439,23 @@ me.actions = {
     ---@return MapPosition?
     go_to_position = function(job, position, find_path)

```

Comment:
this no longer returns anything as far as I can tell -> can remove the annotation